package.json

```diff
@@ -7,8 +7,8 @@
     "build": "npm run build --workspaces",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
-    "lint": "eslint packages/** --ext .ts",
-    "lint:fix": "eslint --fix packages/** --ext .ts"
+    "lint": "eslint packages/** --ext .ts || echo 'No matching files'",
+    "lint:fix": "eslint --fix packages/** --ext .ts || echo 'No matching files'"
```

Comment:
Could you further investigate a way to not fail when there are no matching files? The current construct will never fail in the future, even when it should.